packages/api/src/command/medical/document/document-query.ts

```diff
@@ -85,11 +95,16 @@ export const createQueryResponse = (
 type UpdateResult = {
   patient: Pick<Patient, "id" | "cxId">;
   convertResult: ConvertResult;
+  requestId?: string | undefined | null;
```

Comment:
We use `undefined` to represent a value that's not set, keeping `null` to the "edges" of the app (HTTP, DB, Integrations) - which convert `null` to `undefined` before passing values to domain/logic layers. So I suggest removing `null` from here.